app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -203,7 +203,7 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
 
     final List<LuciBuilder> prodBuilders = await LuciBuilder.getProdBuilders('flutter', config);
     for (LuciBuilder builder in prodBuilders) {
-      tasks.add(newTask(builder.taskName, 'chromebot', <String>['can-update-github'], builder.flaky, 0));
+      tasks.add(Task.chromebot(commitKey, createTimestamp, builder.taskName, builder.flaky));
```

Comment:
maybe only one-line change `builder.flaky` => `builder.flay??false`?
If a builder is missing `isFlaky` flag, we will treat it as `false` by default.

Reply:
What is the default value of `flaky`? It's not documented in https://github.com/flutter/flutter/blob/master/dev/README.md

Reply:
Now the `isFlaky` flag is required by cocoon backend. If for the bad case people forget or have typos, the default value should be `false`. @godofredoc 
Since `isFlaky` is treated as non-optional, not documented on the README.

Reply:
OK